test/parallel/test-https-hwm.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+// Test https highWaterMark
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const https = require('https');
+const fixtures = require('../common/fixtures');
+
+const options = {
+  key: fixtures.readKey('agent1-key.pem'),
+  cert: fixtures.readKey('agent1-cert.pem')
+};
+
+function thrower(er) {
+  throw er;
+}
```

Comment:
You generally don’t need this, uncaught `error` events will be thrown anyway :)

Reply:
done